dashboard/config/initializers/script_preload.rb

```diff
@@ -0,0 +1,2 @@
+# Preload scripts before application fork
+Script.script_cache
```

Comment:
are the tests failing in travis because rake test loads this before it loads the test db schema?